tools/ci_testing/addons_gpu.sh

```diff
@@ -43,19 +29,8 @@ export CUDA_TOOLKIT_PATH="/usr/local/cuda"
 export TF_CUDNN_VERSION="7"
 export CUDNN_INSTALL_PATH="/usr/lib/x86_64-linux-gnu"
 
-# Check if python3 is available. On Windows VM it is not.
-if [ -x "$(command -v python3)" ]; then
-    python3 ./configure.py
-  else
-    python ./configure.py
-fi
```

Comment:
Removing this because GPU isn't available for windows? I'm okay with that but something we'll need to add back once that is complete.

Reply:
Yup, i agree.